sessions/react-with-local-storage/dice/README.md

```diff
@@ -0,0 +1,41 @@
+# React with Local Storage: Dice 🎲
+
+This dice rolling app rolls a die (d6 = six sided die). Now you want to show recent dice rolls in a list and store them in local storage.
+
+## Task
+
+### Lifting State Up
+
+Look at the [`src/App.js`](./src/App.js) and [`src/components/D6Button/index.js`](./src/components/D6Button/index.js) files.
+
+Lift up the `rolls` state into the `App` component. Pass the derived `currentRollValue` variable down to the `D6Button` and wire up a `onClick` handler to call the `handleRoll` function via props. Make sure everything still works as expected.
```

Comment:
As far as I can see, the onClick handler is already existing, one just has to change `onClick={handleRoll}` to `onClick={passedPropName}`. "Wire up" sounds like "write it completely from scratch" to me.

Reply:
Yup, that wording is "a wee bit sloppy".

Reply:
What needs to be done by the students:
- lift state `rolls` and `handleRoll` up to App.js
- pass `rolls` and `handleRoll` to `D6Button` component with appropriate prop names
- pass `rolls` to `History` component

What is not necessary:
- pass `currentRollValue` to `D6Button` because I can keep `currentRollValue` within `D6Button` and I don't need to lift it up.
  - Is this why I don't need to "wire up a `onClick` handler to call the `handleRoll` function via props" (whatever this means exactly).

Please find [my solution here](https://codesandbox.io/s/react-with-local-storage-dice-solution-01-eodd58?file=/src/components/D6Button/index.js).